### PR TITLE
[MIG] account_payment_term_extension: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
- Update README
- Update spanish translation
- Adding changes of https://github.com/OCA/account-invoicing/pull/328

cc @Tecnativa

### CODE REVIEW

File: account_payment_term_extension/i18n/es.po [hunk 1]
```diff
@@ -58,30 +55,30 @@ msgstr "Nombre mostrado"
 #. module: account_payment_term_extension
 #: model:ir.ui.view,arch_db:account_payment_term_extension.view_payment_term_line_form
 msgid "Due Date Computation"
-msgstr ""
+msgstr "álculo de fecha de vencimiento"
```
Comment (pedrobaeza) [inline account_payment_term_extension/i18n/es.po]: Missing initial C

File: account_payment_term_extension/i18n/es_ES.po [hunk 2]
```diff
@@ -1,29 +1,25 @@
 # Translation of Odoo Server.
```
Comment (pedrobaeza) [inline account_payment_term_extension/i18n/es_ES.po]: Remove this file

File: account_payment_term_extension/models/account_payment_term.py [hunk 3]
```diff
@@ -1,12 +1,11 @@
-# -*- coding: utf-8 -*-
 # Copyright 2013-2016 Camptocamp SA (Yannick Vaucher)
 # Copyright 2004-2016 Odoo S.A. (www.odoo.com)
 # Copyright 2015-2016 Akretion
 # (Alexis de Lattre <alexis.delattre@akretion.com>)
 # Copyright 2018 Simone Rubino - Agile Business Group
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-
+from functools import reduce
```
Comment (pedrobaeza) [inline account_payment_term_extension/models/account_payment_term.py]: Why this?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please squash commits by logical set/author

File: account_payment_term_extension/models/account_payment_term.py (hunk 3 above)
Comment (cubells) [reply account_payment_term_extension/models/account_payment_term.py]: reduce function is used here:
https://github.com/OCA/account-invoicing/pull/375/files#diff-d0188f250f37daab85f1680d28b455ccR198
and in python3 does not exist. We have two solutions, import from functools or replace reduce function by a for loop.
I choose import option.

Comment (pedrobaeza) [reply account_payment_term_extension/models/account_payment_term.py]: OK, thanks for the explanations

**Review by pedrobaeza — APPROVED** [review]

**Review by angelmoya — APPROVED** [review]

**Pull request merged**